modules/appeals_api/config/schemas/10182.json

```diff
@@ -46,18 +46,50 @@
       "additionalProperties": false,
       "properties": {
         "homeless":            { "type": "boolean" },
-        "representativesName": { "type": "string", "maxLength": 120 },
-        "address": { "$ref": "#/definitions/nodCreateAddress" }
+        "address":             { "$ref": "#/definitions/nodCreateAddress", "minLength": 5, "maxLength": 165 },
+        "phone":               { "$ref": "#/definitions/nodCreatePhone" },
+        "emailAddressText":    { "$ref": "#/definitions/nodCreateEmail" },
+        "representativesName": { "type": "string", "maxLength": 120 }
       },
-      "required": [ "homeless", "address" ]
+      "required": [ "homeless", "phone", "emailAddressText" ]
     },
 
 
     "nodCreateAddress": {
       "type": "object",
-      "properties": { "zipCode5": { "type": "string", "pattern": "^[0-9]{5}$", "minLength": 5, "maxLength": 5 } },
       "additionalProperties": false,
-      "required": [ "zipCode5" ]
+      "properties": {
+        "addressLine1":            { "type": "string" },
+        "addressLine2":            { "type": "string" },
+        "addressLine3":            { "type": "string" },
+        "city":                    { "type": "string" },
+        "stateCode":               { "$ref": "#/definitions/nodCreateStateCode" },
+        "countryName":             { "type": "string" },
+        "zipCode5":                { "type": "string", "pattern": "^[0-9]{5}$", "minLength": 5, "maxLength": 5 },
+        "internationalPostalCode": { "type": "string" }
+      },
+      "required": [ "addressLine1", "city", "countryName", "zipCode5" ]
+    },
+
+
+    "nodCreatePhone": {
+      "$comment": "the phone fields must not exceed 25 chars, when concatenated",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "countryCode":     { "type": "string", "pattern": "^[0-9]+$", "minLength": 1, "maxLength": 1 },
+        "areaCode":        { "type": "string", "pattern": "^[2-9][0-9]{2}$", "minLength": 3, "maxLength": 3 },
+        "phoneNumber":     { "type": "string", "pattern": "^[0-9]{1,14}$", "minLength": 7, "maxLength": 14 },
```

Comment:
This minLength doesn't follow the pattern and may not be correct for phone numbers outside the US.